generated_code/generate.py

```diff
@@ -176,11 +180,50 @@
   except:
     print('WARNING: ChainForge was not found. Falling back to GemmForge.')
 
+trueOutputDir = os.path.join(cmdLineArgs.outputDir, 'equation')
```

Comment:
Why don't we use `cmdLineArgs.equation` here? This would pave the way for having more than one equation in the same binary?

Reply:
We could definitely do that. Originally, the equations were also templated, by https://github.com/SeisSol/yateto/pull/62 (cf. what was done in #932; that's where most of this PR is from). It just doesn't completely work with the subroutine.cpp file yet.

Reply:
It's changed now to `equation-{equation}-{order}-{precision}`. The `order` part may be slightly overkill (as the matrices are usually pretty similar). But the `equation` partially and the `precision` surely would be necessary, I'd guess